src/org/labkey/test/tests/SampleTypeLinkedStudyExportTest.java

```diff
@@ -0,0 +1,151 @@
+
+        _containerHelper.createProject(STUDY_IMPORT, "Study");
+        _studyHelper.startCreateStudy()
+                .setTimepointType(StudyHelper.TimepointType.VISIT)
+                .createStudy();
+
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        new PortalHelper(getDriver()).addBodyWebPart("Sample Types");
+
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        log("Creating sample types");
+        SampleTypeHelper sampleHelper = new SampleTypeHelper(this);
+        sampleHelper.createSampleType(new SampleTypeDefinition(SAMPLE_TYPE)
+                .setAutoLinkDataToStudy("/" + STUDY_EXPORT)
+                .setFields(List.of(
+                        new FieldDefinition("VisitID", FieldDefinition.ColumnType.VisitId),
+                        new FieldDefinition("Date", FieldDefinition.ColumnType.VisitDate),
+                        new FieldDefinition("ParticipantID", FieldDefinition.ColumnType.Subject))));
+
+        goToProjectHome(STUDY_IMPORT);
+        new PortalHelper(getDriver()).addBodyWebPart("Datasets");
+
+        goToProjectHome(STUDY_EXPORT);
+        new PortalHelper(getDriver()).addBodyWebPart("Datasets");
+    }
+
+    /*
+        Test coverage for : Issue 45238: NPE when importing sample type with auto-link study via pipeline job
+     */
+    @Test
+    public void testImportSampleTypeFolder()
+    {
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE));
+        DataRegionTable samplesTable = DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
+        samplesTable.clickInsertNewRow();
+        setFormElement(Locator.name("quf_Name"), "Blood");
+        setFormElement(Locator.name("quf_VisitID"), "1");
+        setFormElement(Locator.name("quf_Date"), now);
+        setFormElement(Locator.name("quf_ParticipantID"), "P1");
+        clickButton("Submit");
+
+        log("Export the Sample type folder");
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        goToFolderManagement()
+                .goToExportTab();
+        File exportArchive = new ExportFolderPage(getDriver())
+                .includeSampleTypeAndDataClasses(true)
+                .exportToBrowserAsZipFile();
+
+        log("Navigate into the destination folder and import there");
+        goToProjectHome(STUDY_IMPORT);
+        ImportFolderPage.beginAt(this, STUDY_IMPORT)
+                .selectLocalZipArchive()
+                .chooseFile(exportArchive)
+                .clickImportFolder();
+        waitForPipelineJobsToFinish(1);
+
+        goToProjectHome(STUDY_IMPORT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE));
+        DataRegionTable table = DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
+        checker().verifyEquals("Incorrect Columns in imported sample type", Arrays.asList("Name", "Flag", "Visit ID", "Date", "Participant ID", "Linked to Export Study Test Project Study"), table.getColumnLabels());
```

Comment:
Don't hard-code the link column, use the project name constant:
```suggestion
        checker().verifyEquals("Incorrect Columns in imported sample type", Arrays.asList("Name", "Flag", "Visit ID", "Date", "Participant ID", "Linked to " + STUDY_EXPORT + " Study"), table.getColumnLabels());
```